src/PhpCodeExtractor.php

```diff
@@ -11,7 +11,7 @@
 use SplFileInfo;
 use WP_CLI;
 
-class WordPressCodeExtractor extends PhpCode {
+class PhpCodeExtractor extends PhpCode {
```

Comment:
Class should be `final` and protected methods/properties should be private (unless they were already protected in the superclass).